addons/website/controllers/main.py

```diff
@@ -223,6 +223,26 @@ def website_info(self, **kwargs):
         }
         return request.render('website.website_info', values)
 
+    @http.route(['/website/survey/<int:step>', '/website/survey/<int:step>/<int:wid>'], type='http', auth="user", website=True)
+    def website_survey(self, step=1, wid=None, **kwargs):
+        # only website_designer should access the page survey
+        if not request.env.user.has_group('website.group_website_designer'):
+            raise werkzeug.exceptions.NotFound()
+        if wid:
+            website_id = request.env['website'].sudo().browse(wid)
+            if website_id:
+                theme_counts = request.env['ir.module.module'].search_count([('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common'])])
+                if theme_counts < 10:
```

Comment:
Need to check that themes are available on the db.
What I currently do is not a good solution but `if theme_counts == 25:` doesn't seem to be a good solution neither